### ISSUE TITLE
Pick one frontend framework

### ISSUE THREAD

**Issue by taoeffect** [okTurtles/shelterprotocol.net-old#6]
### Problem

The site uses both `preact` and `react`.

### Solution

Pick one. Ricardo recommended `preact` since it is more lightweight than `react`. Study the project and see if it is simple to replace all `react` components with `preact` components.

If so, do it, otherwise replace all `preact` components with `react` since it makes no sense to include both frameworks.

**Comment by SebinSong** [comment]
@taoeffect 
The search-bar feature I worked on before was using [`createProtal()` ](https://react.dev/reference/react-dom/createPortal)react hook so I had to no choice but to implement it using React to minimise the changes. But let me try to switch all existing React stuff to Preact as it seems better performance-wise. But then if that doesn't seem to work, will do the other way around (Turning all existing Preact stuff to React).
